docs/sql-reference/sql-commands/select.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: SELECT
+parent: SQL commands
+grand_parent: SQL reference
+---
+
+# SELECT
+
+The SQL standard for `SELECT` is large and powerful so that you can query data
+with very precise parameters. GlareDB supports most of the [PostgreSQL `SELECT`]
+syntax and will eventually have full support.
+
+## Syntax
+
+```sql
+SELECT [ DISTINCT ]
+    [ * | expression [ [ AS ] output_name ] [, ...] ]
+    [ FROM from_item [, ...] ]
+    [ WHERE condition ]
+    [ GROUP BY [ DISTINCT ] grouping_element [, ...] ]
+    [ HAVING condition ]
+    [ { UNION | INTERSECT | EXCEPT } [ ALL | DISTINCT ] select ]
+    [ ORDER BY expression [ ASC | DESC | USING operator ] [ NULLS { FIRST | LAST } ] [, ...] ]
+    [ LIMIT { count | ALL } ]
+    [ OFFSET start [ ROW | ROWS ] ]
```

Comment:
```suggestion
```

Unlikely we'll support this, so we can just go ahead an remove it.

Reply:
In my testing it worked, but I agree that we shouldn't include it right now

Reply:
nvm, this works